primitives/btor/core/std_add.btor

```diff
@@ -0,0 +1,7 @@
+; BTOR description generated by Yosys 0.33+65 (git sha1 90124dce5, clang 15.0.0 -fPIC -Os) for module std_add.
```

Comment:
We should probably figure out a different place to store these files. The `primitives` folder is shipped with the compiler (using `calyx-primitives` crate) and should not contain these files.

Reply:
seconded!